testing/kokoro/system_tests.sh

```diff
@@ -130,17 +130,22 @@ set -x
 pwd
 date
 
-if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* && -n $GOLANG_SAMPLES_GO_VET ]]; then
-  echo "This test run will run end-to-end tests.";
-  export GOLANG_SAMPLES_E2E_TEST=1
-  ./testing/kokoro/configure_cloudsql.bash;
-fi
-
 export PATH="$PATH:/tmp/google-cloud-sdk/bin";
 if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* ]]; then
   ./testing/kokoro/configure_gcloud.bash;
 fi
 
+# Download and load secrets
+./testing/kokoro/decrypt-secrets.sh
```

Comment:
Nit: it's not really "decrypting" them.

Reply:
Renamed to `pull-secrets.sh` and `store-secrets.sh`.